REQUIREMENTS.md

```diff
@@ -0,0 +1,128 @@
+# Article Requirements
+
+These are the stylistic and informational requirements for articles
+
+## Table of contents
+
+-   [Last Update Date](#last-update-date)
+    -   [Articles With No Sections](#articles-with-no-sections)
+    -   [Articles With Sections](#articles-with-sections)
+-   [Citations](#citations)
+    -   [Whole Article](#whole-article)
+    -   [Sections](#sections)
+
+## Last Update Date
+
+As most of this information is time sensitive, listing the last updated date is important.
+
+Dates need to be in the format `yyyy/mm/dd` for example, `2022/12/25`
+
+### Articles With No Sections
+
+**Use Case:** If you need to date an article without sections
+
+Put a `_Last updated on <date>_` under the metadata table.
+
+**Example**
+
+```markdown
+---
+id: example
+title: Example
+slug: /example
+---
+
+_Last updated on 2020/09/14_
+```
+
+### Articles With Sections
```

Comment:
At what point do you not put a date on a sub section? You have it for h1 and h2, but didn't specify at what point you stop putting last updated dates.

Frankly, I think it's a bit overkill to put "last updated" dates on subsections (h2), and it may even be overkill for h1s as well. At the very least, if we need to have last updated dates for sections, I say we stop at h1s.

Reply:
Im fine with leaving it at h1 until there is a need to go more